CONTRIBUTING.md

```diff
@@ -52,6 +52,155 @@ Please ensure that your notebook runs end-to-end so that it passes our CI.
+https://github.com/aws/amazon-sagemaker-examples/blob/master/aws_marketplace/creating_marketplace_products/Bring_Your_Own-Creating_Algorithm_and_Model_Package.ipynb
+You would modify this file: https://github.com/aws/amazon-sagemaker-examples/blob/master/aws_marketplace/index.rst
+
+
+1. Look for the table of contents directive, `toctree` :
+
+```
+
+.. toctree::
+   :maxdepth: 1
+
+```
+
+1. Add an entry for the new notebook:
+
+```
+
+.. toctree::
+   :maxdepth: 1
+
+   creating_marketplace_products/Bring_Your_Own-Creating_Algorithm_and_Model_Package
+```
+
+#### Adjusting navigation
+
+Some pages have nested title elements that will impact the navigation and depth. The following shows the title, using the top and bottom hash marks (####). Then the single line equals sign (====), then the dashes (----). These are equivalent to H1, H2, and H3, respectively.
+
+```
+################
+AWS Marketplace
+################
+
+Publish algorithm on the AWS Marketplace
+===========================================
+
+Create your algorithm and model package
+----------------------------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   creating_marketplace_products/Bring_Your_Own-Creating_Algorithm_and_Model_Package
+```
+
+You can create further depth by using tilda (~~~~~), asterisk (********), and carrot (^^^^^).
+
+Important: the underline must be at least as long as the title you’re underlining.
+
+#### Adjusting content display
+
+Typically you want to use  `:maxdepth: 1`
+
+You can adjust how much detail from a notebook appears on a page by changing `maxdepth`. Zero and one depth are the same, and these will display just the title. This would be the H1 element for the notebook. Setting this to 2 would display the H2 elements (## Some subtitle) as well.
+
+Sometimes you include topics from other folders on one index page. If you include a subfolder’s index in the TOC using maxdepth of 1, you might just get one entry. So this is an instance where updating maxdepth to 2 would yield a better result.
+
+If more than one entry is displayed for the same notebook, this is because the author of the notebook mistakenly used multiple H1’s. You can see this in the notebooks where they do this:
+
+```
+# Main title
+Some content
+
+## Subtitle
+Some content
+
+# Some other section
+Some content
+```
+
+Then you’ll get a two bullets (the extra “Some other section” when there should only be one for the main title.
+
+#### Troubleshooting
+
+* Each notebook should have at least one section title
+
+> /Users/markhama/Development/amazon-sagemaker-examples/r_examples/r_batch_transform/r_xgboost_batch_transform.ipynb:6: WARNING: Each notebook should have at least one section title
+
+This means the author doesn’t have a title in the notebook. The first markdown block should have a title like `# Some fancy title`. In some cases the author used html tags like <h1>. These render fine on GitHub, but will error in the website build causing the notebook to be skipped.
```

Comment:
Preview the markdown version - I think you need to escape/code-format the `<h1>` because it's making the rest of this sentence header-size.